### PR TITLE
chore(release): rename package canary name

### PR DESCRIPTION
**Pull request by SoonIter**
## Summary

https://github.com/web-infra-dev/rspack/actions/runs/9658721858/job/26641505736

1. `./x version snapshot` rename the canary version 

```
@rspack/binding-canary
@rspack/binding-darwin-arm64-canary
@rspack/binding-darwin-x64-canary
@rspack/binding-linux-x64-gnu-canary
@rspack/binding-win32-x64-msvc-canary
create-rspack-canary
@rspack/core-canary
@rspack/cli-canary
@rspack/dev-server-canary
@rspack/plugin-minify-canary
@rspack/plugin-preact-refresh-canary
@rspack/plugin-react-refresh-canary
@rspack/test-tools-canary
```

2. `npx install-rspack --version 0.7.5-canary-d614005-20240625082730`

https://github.com/rspack-contrib/install-rspack

image

## Checklist

- [x] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

**Comment by SoonIter** [comment]
!canary

File: scripts/debug/overrides-rspack.mjs [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { findWorkspacePackagesNoCheck } from "@pnpm/find-workspace-packages";
+import fs from "node:fs";
+import { getNextName } from "../release/version.mjs";
+
+export async function overrides_rspack_handler(version, options) {
```
[the remark this answers is not in the document]
Comment (SoonIter) [reply scripts/debug/overrides-rspack.mjs]: done in `npx install-rspack --version 0.7.5-canary-d614005-20240625082730`
https://github.com/rspack-contrib/install-rspack

[the remark this answers is not in the document]
Comment (chenjiahan) [reply scripts/debug/overrides-rspack.mjs]: Cool 👍

File: scripts/release/version.mjs [hunk 2]
```diff
@@ -49,10 +58,22 @@ export async function version_handler(version) {
...
+				name: nextName,
+				version: nextVersion
+			};
+		} else {
+			newManifest = {
+				...workspace.manifest,
+				version: nextVersion
+			};
+		}
+		console.log(newManifest.name);
```
Comment (chenjiahan) [inline scripts/release/version.mjs:76]: should we delete this log

Comment (SoonIter) [reply scripts/release/version.mjs:76]: I think it is okay to be preserved
image

**Review by chenjiahan — APPROVED** [review]
👍

**Pull request merged**